--- app.js	2015-10-21 09:25:09.000000000 -0500
+++ new.js	2015-10-21 09:28:20.000000000 -0500
@@ -15,30 +15,26 @@
 var
   http    = require( 'http'    ),
   express = require( 'express' ),
+  morgan = require( 'morgan' ),
+  bodyParser = require( 'body-parser' ),
+  methodOverride = require( 'method-override' ),
+  errorHandler = require( 'errorhandler' ),
 
   app     = express(),
   server  = http.createServer( app );
 // ------------- END MODULE SCOPE VARIABLES ---------------
 
 // ------------- BEGIN SERVER CONFIGURATION ---------------
-app.configure( function () {
-  app.use( express.bodyParser() );
-  app.use( express.methodOverride() );
+  app.use( bodyParser.urlencoded() );
+  app.use( methodOverride() );
   app.use( express.static( __dirname + '/public' ) );
-  app.use( app.router );
-});
-
-app.configure( 'development', function () {
-  app.use( express.logger() );
-  app.use( express.errorHandler({
-    dumpExceptions : true,
-    showStack      : true
-  }) );
-});
 
-app.configure( 'production', function () {
-  app.use( express.errorHandler() );
-});
+  app.use( morgan( 'common' ) );
+  app.use( errorHandler() );
+  //  app.use( errorHandler({
+  //  dumpExceptions : true,
+  //    showStack      : true
+  // });
 
 app.get( '/', function ( request, response ) {
   response.redirect( '/spa.html' );
